src/frontend/containers/block/addBlock.tsx

```diff
@@ -1,10 +1,13 @@
-import React from "react";
-import * as dbTypes from "@db/types";
-import { promiseAsHook } from "@frontend/utils";
-
 import * as note from "./note";
+import * as dbTypes from "@db/types";
+import {
+  // promiseAsHook,
+  observableAsHook,
```

Comment:
just a neurotic naming question: wouldn't the canoncial name rather be `useObservable`?